src/snowflake/snowpark/modin/plugin/extensions/series_overrides.py

```diff
@@ -149,11 +149,6 @@ def autocorr(self, lag=1):  # noqa: PR01, RT01, D200
     pass  # pragma: no cover
 
 
-@register_series_not_implemented()
```

Comment:
So are you saying that all we needed to do was to inherit the modin implementation? Nice!

Reply:
(I guess that's because it's re-written to other supported APIs.)

Reply:
Yep, the modin implementation basically works out of the box.